### ISSUE TITLE
Missing ability to control Emergency/Aux heat

### ISSUE THREAD

**Issue by wirecuttr** [zlangbert/ha-daikinone#25]
Hi, Not sure if I'm missing it, but I can't seem to find a way to turn on Aux heat. I see it in the code, but it doesn't show up as a mode in the UI:
![image](https://github.com/zlangbert/ha-daikinone/assets/40327874/ecd19018-3d17-4c7d-8c56-6cd7cb78a7b7)

I tested the [skyport integration](https://github.com/apetrycki/daikinskyport/tree/master/custom_components/daikinskyport) as well and it also does not have Aux heat showing in the HVAC modes, but with that integration Aux heat can be turned on and off using the service call: Climate: Turn on/off auxiliary heater.

Are you able to add in Aux heat control?

**Comment by zlangbert** [comment]
Yes, definitely want to add support for this. To confirm, you want the ability to set the thermostat to the "emergency heat" mode as shown here right?

<img width="541" alt="image" src="https://github.com/zlangbert/ha-daikinone/assets/228268/5fbf0c20-78f6-4aee-9a1f-87790daa94f9">

**Comment by wirecuttr** [comment]
Yup, that's it. In the physical thermostat UI, it's called 'emergency heat'

**Comment by zlangbert** [comment]
@wirecuttr I just released `0.7.0-beta.0` which has support for setting emergency heat through a preset mode. This works both through the thermostat card in the UI and with the `climate.set_preset_mode` service. 

Try it out and let me know how it goes.

**Comment by wirecuttr** [comment]
Confirmed working for me :)

**Comment by wirecuttr** [comment]
One minor thing... In the preset you labeled it 'Emergency Heat', but Home Assistant uses the more international standard of not capitalizing every word, so for better HA consistency it would be better to label it 'Emergency heat' (lower case h)

**Comment by zlangbert** [comment]
Good call, I updated the string to `Emergency heat`

**Comment by marcobe** [comment]
Question, while you guys are busy with this, would be it be easy to add there 2 features?
- to turn on/off the 'one clean' air feature? 
- to turn on/off air circulation?
It's on my thermostat, and in the Daikin app.

**Comment by wirecuttr** [comment]
Looking through the diagnostics and toggling the feature in my thermostat, Iooks like the One clean feature is triggered by: 'oneCleanFanActive'  true/false
(Unfortunately, I don't know enough to do anything useful with that information)

**Comment by wirecuttr** [comment]
> @zlangbert I have a mix of heat pumps and condenser units with gas furnaces. The payload data "ctSystemCapEmergencyHeat" is true for all, but "modeEmHeatAvailable" seems to track better for me. If the value is true, then "Emergency Heat" is available as a mode in the app. Is there a way to test that value then add to DaikinThermostatCapability?
> 
> @wirecuttr do all of your units have emergency heating available?

Yes. My set up is heat pumps with gas furnaces for emergency heat.

**Comment by zlangbert** [comment]
@otherwish to clarify, you are saying a portion of your systems do not have an emergency heat mode available, and for those that don't, `ctSystemCapEmergencyHeat` is still true but `modeEmHeatAvailable` is false?

**Comment by otherwish** [comment]
> @otherwish to clarify, you are saying a portion of your systems do not have an emergency heat mode available, and for those that don't, `ctSystemCapEmergencyHeat` is still true but `modeEmHeatAvailable` is false?

Correct. I have attached json data from the front and back thermostats. Front thermostat has a cooling only outdoor unit, while back thermostat has a heat pump outdoor unit.

[daikinone-33cd35f95293bc7f1e2ebbf6b87917aa-Front Thermostat-3dc3f67cc1eff7ae16f09b5c57e615e6.json](https://github.com/zlangbert/ha-daikinone/files/14481798/daikinone-33cd35f95293bc7f1e2ebbf6b87917aa-Front.Thermostat-3dc3f67cc1eff7ae16f09b5c57e615e6.json)
[daikinone-33cd35f95293bc7f1e2ebbf6b87917aa-Back Thermostat-87831f09981664f81979edcba99d5942.json](https://github.com/zlangbert/ha-daikinone/files/14481799/daikinone-33cd35f95293bc7f1e2ebbf6b87917aa-Back.Thermostat-87831f09981664f81979edcba99d5942.json)

**Comment by jf-navica** [comment]
May I suggest you add your voice there: https://github.com/home-assistant/architecture/discussions/932
Aux heat should be a heating mode in HA, not a separate entity IMO.
